src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1609,6 +1614,11 @@
     text: az appservice hybrid-connection set-key -g MyResourceGroup --plan MyAppServicePlan --namespace [HybridConectionNamespace] --hybrid-connection [HybridConnectionName] --key-type ["primary"/"secondary"]
 """
 
+helps['appservice plan vnet-integration'] = """
```

Comment:
is this correct ? is this under the group plan? given the hybrid connection one is under appservice??